internal/setup/setup_test.go

```diff
@@ -1950,7 +2037,27 @@ var defaultChiselYaml = `
 `
 
 func (s *S) TestParseRelease(c *C) {
-	for _, test := range setupTests {
+	// Run tests for "archives" field in "v1" format.
+	runParseReleaseTests(c, setupTests)
+
+	// Run tests for "v2-archives" field in "v1" format.
+	v2ArchiveTests := make([]setupTest, len(setupTests))
```

Comment:
Very small suggestion:
```suggestion
	v2ArchiveTests := make([]setupTest, len(setupTests), 0)
```
And then you can forget about the i and use append below instead.

Reply:
Don't you mean the following?

```go
	v2ArchiveTests := make([]setupTest, 0, len(setupTests))
```

Reply:
Changed in 6b7a0be. Thanks.